tests/python/unittest/test_gluon_rnn.py

```diff
@@ -32,49 +31,19 @@ def check_rnn_states(fused_states, stack_states, num_layers, bidirectional=False
     assert len(stack_states) / len(fused_states) == num_layers * directions
 
     fused_states = [state.asnumpy() for state in fused_states]
-    stack_states = [np.expand_dims(state.asnumpy(), axis=0) for state in stack_states]
+    stack_states = [_np.expand_dims(state.asnumpy(), axis=0) for state in stack_states]
     if is_lstm:
         stack_states_h = stack_states[0::2]
         stack_states_c = stack_states[1::2]
-        stack_states = [np.concatenate(stack_states_h, axis=0), np.concatenate(stack_states_c, axis=0)]
+        stack_states = [_np.concatenate(stack_states_h, axis=0), _np.concatenate(stack_states_c, axis=0)]
     else:
-        stack_states = [np.concatenate(stack_states, axis=0)]
+        stack_states = [_np.concatenate(stack_states, axis=0)]
 
     for f, s in zip(fused_states, stack_states):
         assert f.shape == s.shape
         assert_almost_equal(f, s, atol=1e-4, rtol=1e-4)
 
 
-def test_rnn():
-    cell = gluon.rnn.RNNCell(100)
-    inputs = [mx.sym.Variable('t%d_data'%i) for i in range(3)]
-    outputs, _ = cell.unroll(3, inputs)
-    outputs = mx.sym.Group(outputs)
-    assert sorted(cell.collect_params().keys()) == ['h2h_bias', 'h2h_weight',
-                                                    'i2h_bias', 'i2h_weight']
-    assert outputs.list_outputs() == [
-        'rnncell_t0_out_output', 'rnncell_t1_out_output',
-        'rnncell_t2_out_output'
-    ]
-
-    args, outs, auxs = outputs.infer_shape(t0_data=(10,50), t1_data=(10,50), t2_data=(10,50))
-    assert outs == [(10, 100), (10, 100), (10, 100)]
-
-
-def test_lstm():
```

Comment:
Should this test be adapted to testing the front-end infer_shape functions instead?